src/kudu/consensus/consensus_queue.cc

```diff
@@ -1281,6 +1310,10 @@ bool PeerMessageQueue::ResponseFromPeer(const std::string& peer_uuid,
   {
     std::lock_guard<simple_spinlock> scoped_lock(queue_lock_);
 
+    // TODO(mpercy): Handle response from proxy on behalf of another peer.
```

Comment:
I am not sure I understand this comment.

Reply:
Right now we don't handle proxying in a special way when we receive the response at the leader. Maybe in the future we might want to report status in a special way if we know that the proxy is down as opposed to the destination node being down. I wasn't sure if it would help operability to add that so for now I didn't, I simply added a TODO.